### DISCUSSION TITLE
Create a version for the Mac operating system

### DISCUSSION THREAD

**Discussion by mahdi-khoshkhabar** [Tichau/FileConverter#370] [Ideas]
It is a very useful and practical program and I hope that a version for Mac will be made soon

**Comment by Tichau** [comment]
Hello, 
A mac or linux version is not planned for the moment. To do that kind of version we need to do the following work:
1) Port the current solution from .NET Framework to .NET Core.
2) Rewrite the explorer extensions for each needed platform.

I've looked about the first part, to start this process a bit, but unfortunately the .NET Core support of Sharpshell is still in progress (see pull request https://github.com/dwmkerr/sharpshell/pull/331). While it's pending, it is not possible (or we need to rewrite the extension in native language, which is considered here: https://github.com/Tichau/FileConverter/issues/211).

In any case it is a lot of work ^^

**Comment by Tichau** [comment]
related to #400

**Comment by sorenblank** [comment]
Is there any open source alternative for Mac?
